core-lib/Kernel.ns

```diff
@@ -424,6 +437,46 @@ class Kernel vmMirror: vmMirror = Object <: Value (
     public print    = ( '#' print. super print )
   )
 
+  public class Character = String ()
+  ( 
+  ) : (
+    public asciiValue: val = (
+
+    )
+    
+    (*convert to char*)
+    public fromValue: val = (
+      ^vmMirror charFrom: val
+    )
+
+    public value: char = (
```

Comment:
No idea what this is supposed to do. What does it return? an integer? What kind of value? ASCII, utf-8?